src/null-safety/index.md

```diff
@@ -86,10 +76,66 @@ Dart null safety support is based on the following three core design principles:
   fewer bugs, but smaller binaries and faster execution.
 
 
-## Enabling/disabling null safety {#enable-null-safety}
+## Dart 3 and null safety
+
+Dart 3---planned for a mid-2023 release---always has sound null safety.
+Dart 3 will prevent code from running without it.
+
+Packages developed with null safety support will cause issues
+when resolving dependencies:
+
+```
```

Comment:
Consider adding the `terminal` specifier to these so the command and output separation is clear, unless there are major formatting issues.

Reply:
Good idea, done